### DISCUSSION TITLE
Generate array of specified xml element

### DISCUSSION THREAD

**Discussion by harlequin** [mikefarah/yq#1337] [Q&A]
**Describe the bug**
I want to generate always an array of a specific xml element in json output

Version of yq: 4.273

**Input XML**

```xml
<root>
    <item>
        <child>1</child>
    </item>
    <item>
        <child>1</child>
        <child>2</child>
        <child>3</child>
    </item>
</root>
```

**Command**
The command you ran:
`C:/oc/yq -p=x -o=j test.xml`

**Actual behavior**

```json
{
  "root": {
    "item": [
      {
        "child": "1"
      },
      {
        "child": [
          "1",
          "2",
          "3"
        ]
      }
    ]
  }
}
```

**Expected behavior**

```json
{
  "root": {
    "item": [
      {
        "child": [
           "1"
        ]
      },
      {
        "child": [
          "1",
          "2",
          "3"
        ]
      }
    ]
  }
}
```

Is this possible?

**Comment by mikefarah** [comment]
Yes, you can do it like this:

```bash
yq '(.root.item[].child | select(tag != "!!seq")) |= [.]' test.xml -p=x -o=j
```

Explanation:
- We want every `.root.item[].child` to be a sequence, so find all the ones that aren't a sequence `(.root.item[].child | select(tag != "!!seq"))`
- And wrap their value in an array `|= [.]`
